sippy-ng/src/releases/ReleasePayloadPullRequests.js

```diff
@@ -1,30 +1,27 @@
-import { createTheme, makeStyles } from '@material-ui/core/styles'
-import { DataGrid } from '@material-ui/data-grid'
+import { DataGrid } from '@mui/x-data-grid'
+import { makeStyles } from '@mui/styles'
```

Comment:
if I make this `import { makeStyles } from '@mui/styles'`, clicking on "PULL REQUESTS" works; I see `useTheme()` used on line 23 in this file.

Reply:
You are right, sorry, I missed the part about it being on the pull requests page. Fixed!